swan-lake/learn-the-language/language-basics.md

```diff
@@ -473,6 +473,98 @@ function nameToString(Name nm) returns string {
  
 In this way, the ``is`` operator in a condition causes the declared type to be narrowed.
 
+## RegExp Type
```

Comment:
We are adding BBE's for RegExp. Is there any other place where we can explain there stuff?

Reply:
I meant like adding this under a different section like https://ballerina.io/learn/distinctive-language-features/advanced-general-purpose-language-features/.

Reply:
I get that it's kinda a distinct feature of the language. But since it comes under simple values in the spec, shouldn't it come under the basics guide? wdyt?
@hasithaa appreciate your thoughts

Reply:
Guides don't follow the same structure as the spec. Even `decimal` is under distinctive language features/data - https://ballerina.io/learn/distinctive-language-features/data/#decimal-type. Object types is under distinctive language features/Advanced, general-purpose language features while classes are in the basics section.

Reply:
Added this to advance general purpose language features section